packages/core/src/components/Link/Link.types.ts

```diff
@@ -0,0 +1,3 @@
+export type LinkTarget = "_blank" | "_self" | "_parent" | "_top";
+
+export type IconPosition = "start" | "end";
```

Comment:
I think we already have this, maybe in a different PR, we need to make sure we're not exporting it twice, I assume build will fail

Reply:
Are you referring to this?
https://github.com/mondaycom/vibe/pull/2263/files#diff-b41595a7dcb49d9e0d4849662a262434ccdc9423cfc8b24d3e8ae14f49803cf9R1